src/client/pmix_client_get.c

```diff
@@ -594,12 +594,7 @@ static void _getnb_cbfunc(struct pmix_peer_t *pr,
     PMIX_LIST_FOREACH_SAFE(cb, cb2, &pmix_client_globals.pending_requests, pmix_cb_t) {
         if (PMIX_CHECK_NSPACE(proc.nspace, cb->pname.nspace) && cb->pname.rank == proc.rank) {
             pmix_list_remove_item(&pmix_client_globals.pending_requests, &cb->super);
-            if (PMIX_SUCCESS != ret) {
```

Comment:
As stated in the "master" branch version, removing this code block is incorrect.

Reply:
As per the master branch PR, just remove commit https://github.com/openpmix/openpmix/pull/2543/commits/acca23a317fb8e2f61eb675b93fe6c80bee3fb01 and this should be fine.

Reply:
thanks, done.